### ISSUE TITLE
Can not download chrome at Launch()

### ISSUE THREAD

**Issue by zjlx** [ysmood/rod#12]
**Describe the bug**
Can not download chrome at Launch()

**To Reproduce**
Do not set launcher Bin , then it will download chrome.I am from china, the download url will forward taobao.org , but it is not download success.

**Expected behavior**
`opening zip archive for reading: creating reader: zip: not a valid zip file[rod/lib/launcher] Download chromium from: https://npm.taobao.org/mirrors/chromium-browser-snapshots/Win/757680/chrome-win.zip
[rod/lib/launcher] `
I saw the dictionary https://npm.taobao.org/mirrors/chromium-browser-snapshots/Win/, does not have the version 757680,the latest is 737027.

**Rod Version:** v0.26.1

**OS:** Windows

**Comment by ysmood** [comment]
That's a sad story :joy:. Actually you can config the revision like this:

```go
chrome := launcher.NewChrome()
chrome.Revision = "whatever you want"
bin, err := chrome.Get()
kit.E(err)

l := launcher.New().Bin(bin).Launch()
```

**Comment by ysmood** [comment]
Seems like they only keep syncing the revision that `puppeteer.js` uses. Since it's an npm mirror. Maybe we should find a better mirror.

BTW, if you installed the chrome on your local, it should work properly without downloading a new one. So you don't use the default location to install Chrome right?

**Comment by ysmood** [comment]
Anyway, I'll fix it and add a unit test for China. Shouldn't happen again.

**Comment by ysmood** [comment]
Published `v0.26.2`, should work fine now. Please check if you need to reopen it.

**Comment by zjlx** [comment]
> Published `v0.26.2`, should work fine now. Please check if you need to reopen it.

Now it works fine.
